ts/src/bybit.ts

```diff
@@ -1406,27 +1407,26 @@ export default class bybit extends Exchange {
         if (this.options['adjustForTimeDifference']) {
             await this.loadTimeDifference ();
         }
-        let type = undefined;
-        [ type, params ] = this.handleMarketTypeAndParams ('fetchMarkets', undefined, params);
         const promisesUnresolved = [
             this.fetchSpotMarkets (params),
-            this.fetchDerivativesMarkets ({ 'category': 'linear' }),
-            this.fetchDerivativesMarkets ({ 'category': 'inverse' }),
+            this.fetchFutureMarkets ({ 'category': 'linear' }),
+            this.fetchFutureMarkets ({ 'category': 'inverse' }),
+            this.fetchOptionMarkets ({ 'baseCoin': 'BTC' }),
+            this.fetchOptionMarkets ({ 'baseCoin': 'ETH' }),
+            this.fetchOptionMarkets ({ 'baseCoin': 'SOL' }),
```

Comment:
@Dan-krm I would not load them by default because 3 more requests make the markets load considerably slower, and we don't have many users trading options yet.

Reply:
@carlosmiei @Dan-krm There are currently less than 500 contracts for ETH and BTC, and even fewer for SOL. By loading only the first 500 contracts returned by the bybit API, we are able to have all the active contracts for a currency. I think this will considerably reduce markets loading times, unlike today, when all the old contracts are loaded. We could also add a parameter to load all contract histories if necessary.

Reply:
@carlosmiei I checked and Hadevmin is right, if I just add a `(status === 'Trading')` so that only active options are loaded, the load time of loading the option markets by default is almost no different than if I separate them. This also might be a good approach because there's generally only a few methods that check expired options in which case we could automatically enable `loadExpiredOptions` and still leave `loadAllOptions` up to the user for those methods. We could also combine `loadExpiredOptions` and `loadAllOptions` to just have one adjustable setting.

I can add `loadExpiredOptions` or only call fetchOptionMarkets if defaultType is option, or I can do both. I like the approach of having `loadExpiredOptions` and `loadAllOptions` while loading the option markets by default.

Let me know if you think using `loadExpiredOptions` is good enough or if I should still separate calling fetchOptionMarkets to require defining the defaultType, and feel free to adjust this PR as you see fit I'm okay with any of the discussed approaches

Reply:
@Dan-krm Yeah with your latest changes the difference between loading or not the options is smaller according to my quick test, so I think we can keep both options
```
- without options: 
	- `bybit fetchMarkets --sandbox  0.44s user 0.12s system 20% cpu 2.762 total`
	- `node examples/js/cli.js bybit fetchMarkets --sandbox  0.44s user 0.12s system 18% cpu 3.135 total`
- with options
	- `node examples/js/cli.js bybit fetchMarkets --sandbox  0.62s user 0.14s system 14% cpu 5.335 total`
	- `node examples/js/cli.js bybit fetchMarkets --sandbox  0.61s user 0.14s system 14% cpu 5.079 total`
```